h/static/scripts/controllers/character-limit-controller.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+function CharacterLimitController(element) {
+  var counter = element.parentElement.querySelector(
+    '.js-character-limit-counter');
```

Comment:
Ah, my point was not so much about naming as boundaries. The idea is that a controller manages a particular DOM element and its children, so it shouldn't know or depend upon anything of the surrounding structure.